platform/security/encryption/platform-security-encryption-crypter/src/main/java/ddf/security/encryption/crypter/Crypter.java

```diff
@@ -98,7 +100,7 @@ public Crypter() throws CrypterException {
    * @param plainTextValue The value to encrypt.
    */
   public String encrypt(String plainTextValue) throws CrypterException {
-    if (plainTextValue == null || plainTextValue.isEmpty()) {
+    if (isEmpty(plainTextValue)) {
```

Comment:
❓ Do we want to encrypt blank, non-empty strings? Would `encrypt("     ")` be valid input?

Reply:
Great catch. I don't see a reason we should be encrypting blank strings.